pkg/systemd/overview.jsx

```diff
@@ -60,10 +60,26 @@ class OverviewPage extends React.Component {
         this.superuser = cockpit.dbus(null, { bus: "internal" }).proxy("cockpit.Superuser", "/superuser");
     }
 
+    a_new_function(flag) {
+        let foo = 0;
+
+        if (flag) {
+            foo += 1;
+        } else
+            foo = -1; // will not be executed, and we want a report about this
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18469-20230308-121813-a074a05e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)